go/fix-chain/fix_error.go

```diff
@@ -0,0 +1,89 @@
+package fix_chain
```

Comment:
packages should have single word names by convention, no underscores or mixedCaps.